cmd/aries-mobile-sdk/Makefile

```diff
@@ -0,0 +1,78 @@
+# Copyright SecureKey Technologies Inc.
```

Comment:
It would be good if we can link this in the Makefile in root folder, so that we don't need to change the directory while building this.

Reply:
agreed, i'll find a way to link the makefiles together. i expect that there will be many more mobile related targets and don't want to pollute the main makefile.
i'm also avoiding using shell scripts in the mobile section for now because of Windows.